nest_elephant_tvb/translation/nest_to_tvb.py

```diff
@@ -162,54 +161,32 @@ def create_logger(path,name, log_level):
     param = parameters['param_TR_nest_to_tvb']
     store = store_data(path+'/log/',param)
     analyse = analyse_data(path+'/log/',param)
-    level_log = param['level_log']
-    id_spike_detector = os.path.splitext(os.path.basename(path+file_spike_detector))[0]
-    logger_master = create_logger(path, 'nest_to_tvb_master'+str(id_spike_detector), level_log)
-
     # variable for communication between thread
     status_data=[0] # status of the buffer
     initialisation =np.load(param['init']) # initialisation value
     buffer=[initialisation] # buffer of the rate to send it
-
-    ############
-    # Open the MPI port connection for receiver
-    info = MPI.INFO_NULL
-    root=0
-
-    logger_master.info('Translate Receive: before open_port')
-    port_receive = MPI.Open_port(info)
-    logger_master.info('Translate Receive: after open_port: '+port_receive)
-    # Write file configuration of the port
-    path_to_files_receive = path + file_spike_detector
-    fport = open(path_to_files_receive, "w+")
-    fport.write(port_receive)
-    fport.close()
-    pathlib.Path(path_to_files_receive+'.unlock').touch()
-    logger_master.info('Translate Receive: path_file: ' + path_to_files_receive)
-    #########################
-
-    ######################
-    # open for for the sender
-    logger_master.info('Translate SEND: before open_port')
-    port_send = MPI.Open_port(info)
-    logger_master.info('Translate SEND: after open_port : '+port_send)
-    # Write file configuration of the port
-    path_to_files_send = path + TVB_recev_file
-    fport = open(path_to_files_send, "w+")
-    fport.write(port_send)
-    fport.close()
-    pathlib.Path(path_to_files_send+'.unlock').touch()
-    logger_master.info('Translate SEND: path_file: ' + path_to_files_send);sys.stdout.flush()
-    ##############
-    # Wait until connection
-    logger_master.info('Waiting communication')
-    comm_receiver = MPI.COMM_WORLD.Accept(port_receive, info, root)
-    comm_sender = MPI.COMM_WORLD.Accept(port_send, info, root)
-    logger_master.info('get communication and start thread')
-    #############
-
+    
+    ############ NEW Code: old logging code copied here for better overview
+    level_log = param['level_log']
+    id_spike_detector = os.path.splitext(os.path.basename(path+file_spike_detector))[0]
+    logger_master = create_logger(path, 'nest_to_tvb_master'+str(id_spike_detector), level_log)
     logger_receive = create_logger(path, 'nest_to_tvb_receive'+str(id_spike_detector), level_log)
     logger_send = create_logger(path, 'nest_to_tvb_send'+str(id_spike_detector), level_log)
+    ############# NEW Code end
+    
+    ############ NEW Code: FAT END POINT for MPI and new connections
+    ### contains all MPI connection stuff for proper encapsulation
+    ### TODO: make this a proper interface
+    import nest_elephant_tvb.translation.FatEndPoint as FEP
+    
+    path_to_files_receive = path + file_spike_detector
```

Comment:
This feels fragile. I would suggest to use a clear path operation. and not use string concatenations

Reply:
```suggestion
    path_to_files_receive = path + file_spike_detector # TODO: use proper path operations
```
Added a TODO, this is related to one of the above suggestion to use the 'os' module or similar in general.